internal/core/graphics/wgpu_26.rs

```diff
@@ -199,10 +199,11 @@ pub fn any_wgpu26_adapters_with_gpu(requested_graphics_api: Option<RequestedGrap
         }
         Some(_) => return false,
     };
+    let allow_cpu = std::env::var("SLINT_WGPU_CPU").map(|v| v == "1").unwrap_or(false);
```

Comment:
You don't have to check the value, you can just check if the variable is set, with `.is_ok()`.